### PR TITLE
[eclipse/xtext-xtend#818] Add classgraph to target platform

### PR DESCRIPTION
**Pull request by szarnekow**
[eclipse/xtext-xtend#818] Add classgraph to target platform

### CODE REVIEW

File: releng/org.eclipse.xtext.redist.feature/feature.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@ http://www.eclipse.org/legal/epl-v10.html Description here.
       <import plugin="com.google.inject" version="3.0.0" match="greaterOrEqual"/>
       <import plugin="org.objectweb.asm" version="7.1.0" match="compatible"/>
       <import plugin="javax.inject" version="1.0.0" match="greaterOrEqual"/>
+      <import plugin="io.github.classgraph" version="4.8.35" match="compatible"/>
```
Comment (cdietrich) [inline releng/org.eclipse.xtext.redist.feature/feature.xml]: aint that needed for all features?

Comment (szarnekow) [reply releng/org.eclipse.xtext.redist.feature/feature.xml]: I thought redist was included in the others. Let me check.

Comment (szarnekow) [reply releng/org.eclipse.xtext.redist.feature/feature.xml]: Updated. Does that make more sense?

**Comment by cdietrich** [comment]
@szarnekow anything that speaks against merging?

**Comment by szarnekow** [comment]
> @szarnekow anything that speaks against merging?

No, nothing. It just took ages to get a green build.

**Pull request merged**